dlp/snippets/src/main/java/dlp/snippets/DeIdentifyTableBucketing.java

```diff
@@ -36,6 +36,10 @@
 
 public class DeIdentifyTableBucketing {
 
+  public static void main(String[] args) throws IOException {
```

Comment:
Please choose one or the other on using the main method versus the method without args. Please update for all samples.

Reply:
Updated the changes, please check.